backend/src/Entity/StoreOwnerProfileEntity.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+namespace App\Entity;
+use App\Repository\Profile\StoreOwnerProfileEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: StoreOwnerProfileEntityRepository::class)]
+class StoreOwnerProfileEntity
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\Column(type: 'string', length: 255, nullable: true)]
+    private $storeOwnerID;
+
+    #[ORM\Column(type: 'string', length: 255, nullable: true)]
+    private $storeOwnerName;
+
+    #[ORM\Column(type: 'string', length: 255, nullable: true)]
+    private $image;
```

Comment:
please keep in mind this will be 0neToMany relation with image table but it's okay for now